client/my-sites/google-my-business/stats/chart.js

```diff
@@ -44,6 +47,10 @@ class GoogleMyBusinessStatsChart extends Component {
 		this.state = {
 			transformedData: this.transformData( props.data ),
 		};
+
+		if ( ! props.data ) {
+			props.requestGoogleMyBusinessStats( props.siteId, props.statType, props.interval, 'total' );
+		}
```

Comment:
We should run side effects in `componentDidMount()` rather than [in the constructor](https://reactjs.org/docs/react-component.html#constructor).

Also we don't have to specify `total` here since that's the default value.